AppCenterAuth/AppCenterAuthTests/MSAuthTests.m

```diff
@@ -32,7 +32,12 @@
 
 @interface MSAuth (Test)
 
+@property(atomic) BOOL signInShouldWaitForConfig;
+
 - (void)configAuthenticationClient;
+- (void)ifSignInIsInProgressCompleteWithErrorCode:(NSInteger)errorCode
```

Comment:
Isn't `ifSignInIsInProgressCompleteWithErrorCode:` already removed from MSAuth? You want to cleanup the tests.